types/ember__object/test/get-set.ts

```diff
@@ -0,0 +1,10 @@
+import { get, set } from '@ember/object';
+
+const basicPojo = { greeting: 'hello' };
+
+get(basicPojo, 'greeting'); // $ExpectType string
```

Comment:
I know `types/ember/test/ember-module-tests.ts` tests that a bad key returns `unknown`, but, IMHO we should also test that here in `ember__object`:
```ts
get(basicPojo, 'salutation'); // $ExpectType unknown
```

Reply:
Excellent point. Thank you!